CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
-cmake_minimum_required(VERSION 3.12...3.23)
+# Please keep 3.14 as minimum requirement for building on OPAL-RT targets
+cmake_minimum_required(VERSION 3.14...3.23)
```

Comment:
I don't understand why this should be necessary. 3.14 is between 3.12 and 3.23.
This is a change I would really rather avoid, because it for sure breaks compiling on certain targets.

Reply:
3.14 is the first version which supports `FetchContent_MakeAvailable` and hence simplifies the `cmake/Fetch*.cmake` CMake files a lot.

At the same time, its the lowest version of CMake which is shipped by the supported Ubuntu, Debian, Rocky and Fedora Linux distros.

Actually 3.14 is the version shipped with OPAL RT-Linux. The other Linux distros ship at least with 3.16

Reply:
ok sounds fine then